docker/solr/schema.xml

```diff
@@ -1,282 +1,282 @@
-<?xml version="1.0" encoding="UTF-8" ?>
-<schema name="packagist" version="1.4">

```

Comment:
Why are you changing all lines in these files ? Are you changing line endings ?

Reply:
It was my IDE, it changed line separators.

Reply:
if the file has CRLF in it currently, normalizing to LF is fine (we should have LF)